docker/test-images/zipkin-opensearch2/start-opensearch

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# Copyright The OpenZipkin Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# ENTRYPOINT script that starts OpenSearch
+#
+# This intentionally locates config using the current working directory, in order to consolidate
+# Dockerfile instructions to WORKDIR
+set -eu
+
+# Configure the Docker HEALTHCHECK
+export HEALTHCHECK_PORT=9200
+export HEALTHCHECK_PATH=/_cluster/health
+
+# Set tmp folder
+OPENSEARCH_TMPDIR=`java -cp 'classes:lib/*' org.opensearch.tools.launchers.TempDirectory`
```

Comment:
we are setting this to /tmp as an env variable in dockerfile to /tmp. do you want to always overwrite it like this?

Reply:
or does TempDirectory consider an existing OPENSEARCH_TMPDIR.. if so, maybe mention in the comment like below on OPENSEARCH_JAVA_OPTS

Reply:
Good question, sp `TempDirectory` uses `java.io.tmpdir`  and pre-creates the `opensearch` folder there